templates/cluster-template-oci-addons.yaml

```diff
@@ -80,13 +80,13 @@ spec:
     spec:
       imageId: "${OCI_IMAGE_ID}"
       compartmentId: "${OCI_COMPARTMENT_ID}"
-      shape: "${OCI_SHAPE}"
+      shape: "${OCI_CONTROL_PLANE_SHAPE=VM.Standard.E4.Flex}"
       shapeConfig:
-        ocpus: "${OCI_SHAPE_OCPUS}"
-        memoryInGBs: "${OCI_SHAPE_MEMORY_IN_GBS}"
+        ocpus: "${OCI_CONTROL_PLANE_SHAPE_OCPUS=1}"
+        memoryInGBs: "${OCI_CONTROL_PLANE_SHAPE_MEMORY_IN_GBS=16}"
```

Comment:
- [ ] Remove the `=16` and test that blank works. If it doesn't look at modifying the MachineController.

Reply:
@shyamradhakrishnan the "missing ENV" error is coming from `clusterctl generate cluster` https://github.com/kubernetes-sigs/cluster-api/blob/main/cmd/clusterctl/client/yamlprocessor/simple_processor.go#L108 I think we wither have to set a default, or pull the ENV out of the yaml.

Reply:
For now we are going to pull the `OCI_CONTROL_PLANE_SHAPE_MEMORY_IN_GBS` ENV and document how to override if needed.

Reply:
Why?